src/hooks/usePantry.ts

```diff
@@ -181,9 +181,52 @@ function entry({ project }: { project: string }): Entry {
     const versions = dir.join("versions.txt")
     return { dir, yml, versions }
   }
-  throw new TeaError('not-found: pantry: package.yml', {project})
-}
+  //check for fuzzy string
+  const closestPkg = getClosestPackageSuggestion(project, prefix);
+  closestPkg.then(function(data){
+    console.log("did you mean", {data}, "?")
+  });
+  throw new TeaError('not-found: pantry: package.yml', {project}, )
 
+}
+async function getClosestPackageSuggestion(orgPkg:string, projectPath:Path): Promise<string>{
+  let closestPkg = '';
+  let minDistance = Infinity;
+  const pkgList = [];
+  for await (const data of projectPath.ls()){
```

Comment:
I _think_ `ls()` from `usePantry.ls.ts` will work here.

Reply:
Yes, it works. Thanks.